homeassistant/components/netatmo/__init__.py

```diff
@@ -61,9 +106,88 @@ def setup(hass, config):
         for component in 'camera', 'sensor', 'binary_sensor', 'climate':
             discovery.load_platform(hass, component, DOMAIN, {}, config)
 
+    if config[DOMAIN][CONF_WEBHOOKS]:
+        webhook_id = hass.components.webhook.async_generate_id()
```

Comment:
We shouldn't call an async function from a sync context. Probably use `hass.add_job` to schedule the async function.